### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by louismarquenet**

### CODE REVIEW

**Comment by louismarquenet** [comment]
Trying to figure out how to show the rates for each villa.  Knowing that there is different rates according to the season. Then our client must be able do edit and add a period and a price

**Review by AhmadSAFAR — APPROVED** [review]
ok

File: src/Entity/Rate.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+    /**
+     * @ORM\Column(type="decimal", precision=10, scale=0)
+     */
+    private $price;
+
+    /**
+     * @ORM\Column(type="date")
+     */
+    private $year;
```
Comment (RomainClair) [inline src/Entity/Rate.php:38]: Isn't the year in the fisrtPeriodDate ? (and in the second...)

**Review by RomainClair — CHANGES_REQUESTED** [review]
stop make:crud everything !

**Pull request closed without merging**